docs/docs/command-line-flags.md

```diff
@@ -69,7 +69,15 @@ The core functionality flags can be also set by environment variable `MARATHON_O
 * `--max_tasks_per_offer_cycle` (Optional. Default: 1000): Launch at most this number of tasks per Mesos offer cycle.
   A larger value speeds up launching new tasks.
   Yet, choosing a too large value might overwhelm Mesos/Marathon with processing task updates.
-* `--mesos_role` (Optional. Default: None): Mesos role for this framework.
+* `--mesos_role` (Optional. Default: None): Mesos role for this framework. If set, Marathon receives resource offers
+    for the specified role in addition to resources with the role designation '*'.
+* `--default_accepted_resource_roles` (Optional. Default: all roles): Default for the `"defaultAcceptedResourceRoles"`
+    attribute as a comma-separated list of strings. All app definitions which do not specify this attribute explicitly
+    use this value. This defaults to all roles
+    for which this Marathon instance is configured to receive offers (see `--mesos_role`). Changing this configuration
```

Comment:
I think it is better to describe _when_ this default  applies and what the consequence is. 
I would omit hints regarding restarts in usage info.